joern-cli/frontends/jssrc2cpg/src/main/scala/io/joern/jssrc2cpg/astcreation/AstForFunctionsCreator.scala

```diff
@@ -30,11 +30,20 @@ trait AstForFunctionsCreator {
     createBabelNodeInfo(param) match {
       case rest @ BabelNodeInfo(BabelAst.RestElement) =>
         val paramName = rest.code.replace("...", "")
+        val tpe       = typeFor(rest)
```

Comment:
what does tpe stand for? maybe writing it out is a better idea

Reply:
Stands for type which is a reserved keyword in Scala.

Reply:
ohhh, ok. makes sense but since i couldn't tell what it means pbb others will hit the same issue. how about _type or smth?

Reply:
i'd vote for leaving it as `tpe`, we really use it in a lot of places already, and not just us